src/FilterFormatManager.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_migration;
+
+use Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException;
+use Drupal\Component\Plugin\Exception\PluginNotFoundException;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\filter\Entity\FilterFormat;
+use Drupal\filter\FilterPluginCollection;
+
+/**
+ * Class FilterFormatManager.
+ *
+ * Provides filter format related methods.
+ *
+ * @package Drupal\oe_migration
+ */
+class FilterFormatManager {
```

Comment:
Maybe worth creating interfaces for the services such as these so that they can be swappable?

Reply:
**Status:** Fixed
The `FilterFormatManagerInterface` has been created and implemented.